### PR TITLE
Pass strength

### PR DESCRIPTION
**Pull request by jhrbva**
To test this PR:

1. Go to the signup form
2. Enter all the information and when you get to password, add less than 8 characters.
You should see a error message like this:
Screen Shot 2020-03-07 at 1 56 50 PM

3. Now enter 8 or more characters but no numbers, or upper case letters. 
You should see and error message like this:
Screen Shot 2020-03-07 at 1 57 10 PM

4. Enter a password that fits the password strength specifications, you should stop seeing the errors.

Note: a new user is **not** being created yet and after you are done with the form you should **not** be redirected anywhere. This functionality is happening in another ticket.

### CODE REVIEW

**Comment by Quetourah** [comment]
@jhrbva, this looks good to me so far. However, do we want to also require users to include a character like "#$^&)("? Any character that won't allow sql injection. Right now, users can add a character but there is no warning message. just making sure we're not overlooking something.

**Comment by jhrbva** [comment]
@Quetourah this was a deliberate decision. I was with Khristian and Connie when I was building it and we figured that min 8 characters, 1 number, 1 upper case, and 1 lower case was enough for our application. But if you feel that we should add the special character requirement to passwords I can definitely go ahead and add that to this PR.

File: src/utilities/formValidation.js [hunk 1]
```diff
@@ -26,3 +30,26 @@ export const validatePhone = value => {
...
+export const validatePasswordLength = value => {
+	console.log('validatePasswordLength being run');
+	let error;
+	if ((error = required(value)));
+	else if (value.length < 8) error = invalidPasswordLength;
+	return error;
+};
+
+export const validatePasswordStrength = value => {
+	console.log('validatePasswordStrength being run');
```
Comment (kcode20) [inline src/utilities/formValidation.js]: Can we remove these `console.log`'s?

File: src/utilities/formValidation.js [hunk 2]
```diff
@@ -26,3 +30,26 @@ export const validatePhone = value => {
 	else if (!phoneRegex.test(value)) error = invalidPhone;
 	return error;
 };
+
+export const validatePasswordLength = value => {
+	console.log('validatePasswordLength being run');
```
Comment (kcode20) [inline src/utilities/formValidation.js]: This one too.

**Pull request merged**